platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolveConfigurationDependenciesBuildOperationIntegrationTest.groovy

```diff
@@ -464,7 +462,8 @@ class ResolveConfigurationDependenciesBuildOperationIntegrationTest extends Abst
         failedResolve.assertFailurePresent(failure)
         def op = operations.first(ResolveConfigurationDependenciesBuildOperationType)
         op.details.configurationName == "compile"
-        op.failure == null
+        op.failure != null
```

Comment:
🤔 If the failure on the operation is non-null, I think that Develocity will not show the configuration in the UI. Can you check by creating a scan from a Gradle distribution with your changes?

Reply:
Using this buildscript:

```
plugins {
    id("java-library")
}

repositories {
    mavenCentral()
}

configurations.runtimeClasspath {
    resolutionStrategy.failOnVersionConflict()
}

dependencies {
    implementation("com.google.guava:guava:33.4.0-jre")
    implementation("com.google.guava:guava:33.0.0-jre")
}

tasks.register("resolve") {
    val files = configurations.runtimeClasspath.get().incoming.files
    doLast {
        println(files.map { it.name })
    }
}
```



https://develocity.grdev.net/s/kaugollbukvow

Reply:
The build scan's dependencies tab was empty. 

This is with changes to make the build operation failure unset.

https://develocity.grdev.net/s/zqfp42doedk4w

The dependencies are shown. 

I'll remove the code that sets the failure